src/ide/code_actions/mod.rs

```diff
@@ -88,6 +89,9 @@ fn get_code_actions_for_diagnostics(
             "E0003" => fill_struct_fields::fill_struct_fields(db, node.clone(), params)
                 .map(|result| vec![result])
                 .unwrap_or_default(),
+            "E0004" => fill_trait_members::fill_trait_members(db, node.clone(), params)
```

Comment:
What happens if it doesn't return anything? Nothing happens? 
I think that might be weird UX for the users:

1. You click "fill trait members"
2. Nothing happens 
3. You wait, or re-run and still nothing

If it's the case, i think it should be handled in a different PR (create an issue please 🙏 ) since it looks like other actions follow this pattern too

Reply:
>What happens if it doesn't return anything?

Then no code action is shown

```rs
                .map(|result| vec![result])
                .unwrap_or_default(),